council/council.html

```diff
@@ -48,7 +48,7 @@ <h4>Nearby</h4>
       <p>This document supplements the Process Document by providing best practices to resolve and decide Formal Objections.</p>
 
       <p>In this document, <dfn id="resolving">resolving</dfn> a Formal Objection means finding a solution that has no objections. <dfn id="deciding">Deciding</dfn> means 
-        giving a yes/no answer to the question of whether an objection should be <dfn id="overrule">overruled</dfn> (i.e., no) or <dfn id="sustained">sustained</dfn> (i.e., yes).
+        giving a yes/no answer to the question of whether an objection should be <dfn id="overrule">overruled</dfn> (i.e., no) or <dfn id="sustained">sustained</dfn> (i.e., yes); <a href="#council">Council</a> refers to the ephemeral W3C Council created to make a <a href="deciding">decision</a> on a Formal Objection, Councils made of different members may exist at the same time to handle distinct Formal Objections.
```

Comment:
Conceptually, if more than one Decision is subject to an FO that is going to Council, each Decision has a distinct Council whose membership is likely to, but does not necessarily, overlap.

Councils are for handling Decisions, not Formal Objections. For example, if the Decision is "transition to next maturity level" and there are 3 FOs, there would be one Council for the Decision.

I don't think this proposed wording captures those nuances quite correctly.

Reply:
I had the same thing in mind, but indeed the wording is not good, as it can be interpreted as "one Council per FO" which is wrong. Thanks!

Reply:
```suggestion
        giving a yes/no answer to the question of whether an objection should be <dfn id="overrule">overruled</dfn> (i.e., no) or <dfn id="sustained">sustained</dfn> (i.e., yes); <a href="#council">Council</a> refers to the ephemeral W3C Council created to make a <a href="deciding">decision</a> on one or more Formal Objections, Councils made of different members may exist at the same time to handle distinct decisions.
```

Reply:
I believe this proposal would resolve the ambiguity.

Reply:
Minor punctuation and language tweak of @plehegar's suggestion.
```suggestion
        giving a yes/no answer to the question of whether an objection should be <dfn id="overrule">overruled</dfn> (i.e., no) or <dfn id="sustained">sustained</dfn> (i.e., yes); <a href="#council">Council</a> refers to the ephemeral W3C Council created to make a <a href="deciding">decision</a> on one or more Formal Objections. Councils comprising different members may exist at the same time to handle distinct decisions.
```